CMakeLists.txt

```diff
@@ -13,6 +13,13 @@ zeek_plugin_bif(src/af_packet.bif)
 zeek_plugin_dist_files(zeekctl/af_packet.py README COPYING VERSION)
 zeek_plugin_end()
 
+if ( AF_PACKET_HARDWARE_OFFLOAD )
```

Comment:
Agree with your comment, probably not useful if there's a variable and no need for the configure flag.

The name's also too generic not mentioning checksum, but anyway, would suggest to just remove.